### ISSUE TITLE
use with VSG?

### ISSUE THREAD

**Issue by mousimo** [osqzss/gps-sdr-sim#264]
Is there any reason why this tool couldn't be used to simply generate the IQ data needed to supply a vector signal generator? That is, to not drive an SDR at all, but rather to drive a VSG.

**Comment by terrymarkovich** [comment]
Yes, you can do that. The basic gpssim binary just generates interleaved 16 bit IQ.
